src/redux/user/userActions.js

```diff
@@ -0,0 +1,78 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+export const registerUser = createAsyncThunk(
+  'user/register',
+  async ({ firstName, email, password }, { rejectWithValue }) => {
+    const config = {
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    };
+
+    try {
+      await axios.post(
+        '/api/user/register',
```

Comment:
Please consider using env vars or consts for endpoint urls.
```suggestion
        `{process.env.API_URL}/user/register`,
```

Reply:
I’m using proxy in package.json

Reply:
Changed, this will be better)

Reply:
We can also import axios in index.js and set default url to global axios object:

` _index.js_

import axios from "axios";
axios.defaults.baseURL = process.env.API_URL;`